src/Pages/Detail/Detail.js

```diff
@@ -0,0 +1,121 @@
+import React from 'react';
+import { withRouter } from 'react-router-dom';
+import API from '../../config';
+import Goods from './DetailSections/Goods';
+import Categories from './DetailSections/Categories';
+import Popup from '../../Components/ShoppingModal/Popup';
+import Login from '../Login/Login';
+import './Detail.scss';
+
+class Detail extends React.Component {
+  state = {
+    goods: {},
+    isModalOpen: false,
+    isLogin: false,
+    content: {},
+  };
+
+  handleModal = (e, amount) => {
+    const { name } = e.target;
+    const { isLogin, goods } = this.state;
+    const { id, korean_name, price } = goods;
+
+    const order = { id, korean_name, amount, price };
+
+    if (!isLogin) {
+      this.addOrder(order);
+      if (name === 'buy') {
+        this.props.history.push('/payment');
+        return;
+      }
+      this.setState({ isModalOpen: true });
+      return;
+    }
+
+    if (name === 'buy' || name === 'cart') {
+      this.setState({ isModalOpen: true });
+      return;
+    }
+
+    this.setState({ isModalOpen: false });
+  };
+
+  addOrder = data => {
+    const token = localStorage.getItem('access_token');
+    const resource = `/orders/cart`;
+
+    fetch(API.detail + resource, {
+      headers: {
+        Authentication: token,
```

Comment:
```suggestion
        Authorization: token,
```
키 값이 잘못 된 것 같습니다! 다음 문서 참고해주세요
https://developer.mozilla.org/ko/docs/Web/HTTP/Headers/Authorization

Reply:
넵 해당 부분은 문서 확인하도록 하겠습니다.